packages/docs-site/docs/ref/constraints.md

```diff
@@ -128,43 +128,6 @@ Previously, we've talked about how we convert everything to zero-based inequalit
 
 One common operation is to access the parameter of a shape via `shapeName.propertyName.contents`, which will return a `ad.Num`. For example, if you have a circle `c` as input, and you want its radius, `c.r.contents` will give you something like `5.0` (of type `ad.Num`).
 
-## Constraints Example: minSize & maxSize
```

Comment:
Thanks for updating the docs too! In this case it's probably better to change this example to another one. Can you open an issue so we remember to do this later?

Reply:
This whole page was outdated since it still used the older shape-tuples (`[shapeType, props]`) which doesn't exist anymore since the shape-type refactoring. Since one of our goals is to make it easier for outsiders to contribute library functions (I think), I will write up an issue to update this page.

Reply:
See #1488

Reply:
Looks good. Thanks!